src/config.js

```diff
@@ -28,7 +28,9 @@ const jestConfig = {
     transform: {
         '^.+\\.(js|html|css)$': require.resolve('@lwc/jest-transformer')
     },
-    transformIgnorePatterns: ["/node_modules/(?:(?!lightning-stubs.*(js|html|css)))*$"],
+    transformIgnorePatterns: [
```

Comment:
_I don't know if the following is a problem for sfdx-lwc-jest_

If @salesforce/sfdx-lwc-jest is nested inside another node_modules folder like `/node_modules/foo/node_modules/@salesforce/sfdx-lwc-jest/src/lightning-stubs/avatar/avatar.js`, this regex will match and the stubs will not be transformed.